### DISCUSSION TITLE
資工系都該知道的遊戲-Factorio

### DISCUSSION THREAD

**Discussion by TobywithDino** [1101-GS4719-C/Python-Community-GS4719-C#1887] [General]
我超愛玩電腦遊戲，也喜歡程式語言。
要向大家介紹一款遊戲
# Factorio
你意外墜落，目標是要合成足夠的零件，合成出一架火箭，讓你離開這座一堆噁爛蟲子的星球
沒錯，有一堆噁爛蟲子，雖然他主打**自動化**你的工廠，同時你也要發展武器相關工業，抵禦原住民的攻擊!!!

在這款遊戲你可以創造自己的義大利麵
或是打造龐大有序的工廠!!
快去買來玩看看吧!!!

**Comment by woohsin** [comment]
這個遊戲聽起來非常有趣和多元化！從合成零件到打造火箭，再到發展武器相關工業，遊戲內容相當豐富和具有挑戰性。同時，還有創造自己的義大利麵和打造有序工廠等多樣的遊戲元素，應該能夠吸引到不同類型的玩家。

透過遊戲中的工業發展和武器防禦，玩家可以體驗到管理和戰略思維的樂趣，同時也可以挑戰自己的創造力和工程能力。而遇到原住民的攻擊可能會增加遊戲的挑戰度，讓玩家在應對危機的同時，找到解決問題的方法。
